recipes/shrinkwrap/build.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -e
+set -x
+
+cmake \
+  -DBUILD_SHARED_LIBS:BOOL=ON \
+  -DCMAKE_PREFIX_PATH:PATH=${PREFIX} \
+  -DCMAKE_INSTALL_PREFIX:PATH=${PREFIX} \
+  -DCMAKE_INSTALL_INCLUDEDIR=include \
+  -DCMAKE_INSTALL_LIBDIR=lib \
+  -DCMAKE_BUILD_TYPE=Release \
+  ${SRC_DIR}
+
+make -j${CPU_COUNT}
+make install
+make test
```

Comment:
Do we have that for Windows as well? (Sorry I did not mention that in the first message.)